jekyll/index.html

```diff
@@ -84,25 +84,24 @@ <h2 class="major">{% t 'section_projects.title' %}</h2>
         </div>
     </section>
 
-    <!-- Najnowsze newsy - kilka=4-->
     <section id="news" class="wrapper style3">
         <div class="inner">
             <h2 class="major">{% t 'section_news.title' %}</h2>
             <p>{% t 'section_news.text' %}</p>
             <section class="features">
-                {% for post in site.posts %}
+            {% for post in site.posts %}
                 <article>
                     {% if post.post_img_url != "" %}
-                    <a href="{{ post.url }}" class="image"><img src="{{ post.post_img_url }}" alt=""/></a>
+                        {% picture halfpage {{ post.post_img_url }} --link {{ post.url }} --a class="image" %}
                     {% endif; %}
                     <h3 class="major">{{ post.date | date: "%Y-%m-%d" }} {{ post.title }}</h3>
                     {{ post.excerpt }}
                     <a href="{{ post.url }}" class="special">{% t 'read_more' %}</a>
                 </article>
                 {% if forloop.index == 4 %}
-                {% break %}
+                    {% break %}
                 {% endif %}
```

Comment:
tego dora to chyba lepiej w ogóle przenieść wyżej, `for post in site.posts, index <= 4` czy jak to się tam robi w tym jekyllu

Reply:
fixed
CC @danielskowronski